### ISSUE TITLE
Installation blocked by windows

### ISSUE THREAD

**Issue by renan905** [microsoft/vscode#104458]
- VSCode Version: --
- OS Version: Windows 10 Pro - 2004, build 19041.450

I am trying to reinstall VSCode after a Windows reset but, Windows are blocking the installation by a security warning (see print warning below). User Installer, System Installer, and Insiders Edition Installer present the same problem.

Yes, I have downloaded the files (multiple times) from the official site.

![image](https://user-images.githubusercontent.com/20269506/89955002-7a6c7a00-dc08-11ea-9b3f-3a16c6d74995.png)

**Comment by aitc-h** [comment]
The authentic version of VS Code should be signed by Microsoft. Did you download it from the official source (https://code.visualstudio.com/)?

**Comment by renan905** [comment]
> The authentic version of VS Code should be signed by Microsoft. Did you download it from the official source (https://code.visualstudio.com/)?

Yes, i have downloaded from the official source

**Comment by aitc-h** [comment]
Does this happen with other application that you have tried?

**Comment by renan905** [comment]
No, this only happens with vscode

**Comment by ConradSollitt** [comment]
@renan905 Perhaps your system is compromised and the download was replaced by another program. Or another issue if your system is compromised could be something related to any program being installed.

Try this command from `cmd.exe` and compare to the print screen below:
`CertUtil -hashfile VSCodeUserSetup-x64-1.47.3.exe SHA256`

![image](https://user-images.githubusercontent.com/57777521/89962326-3de55200-dbf9-11ea-9079-604e5d9bcc82.png)

If you right-click and select properties on the downloaded EXE does it show a [Digital Signatures] tab? Although this was taken from Win 7 and I think you are on Win 10.

![image](https://user-images.githubusercontent.com/57777521/89962497-b5b37c80-dbf9-11ea-9f60-432db380ce1e.png)

**Comment by renan905** [comment]
@ConradSollitt the file SHA seems to be the same.

![image](https://user-images.githubusercontent.com/20269506/89963116-25872e80-dc1d-11ea-864a-69c13b81e5ca.png)

The [Digital Signatures] tab is also available in Windows 10, the following imagen shows the certificate details.

![image](https://user-images.githubusercontent.com/20269506/89963286-90386a00-dc1d-11ea-9ded-976ffff64caa.png)

That's the first time I have problems relatable with certificates. Maybe the Windows reset (performed by the windows build-in tool) has broken something, but that fact that other applications don't cause any problem quite confuses me.

**Comment by renan905** [comment]
@ConradSollitt About the system maybe has been a compromise, I have run: 

`DISM.exe /Online /Cleanup-image /Restorehealth`
`sfc /scannow`

If I'm not wrong this should identify problems with my Windows build/image, and everything was fine

**Comment by renan905** [comment]
**Update:** I still can't install the vscode from the official site release, however, I was able to install using chocolatey and is working 100% fine, no problems at all

**Comment by ConradSollitt** [comment]
@renan905 Sorry, I didn't get time to research this today. Good that you got it working with chocolatey at least.

Very strange error, would be nice to solve. Sounds like it's more of an issue with Windows now rather than your system being hacked or an issue with VS Code. It reminds me once a few years ago where I had a signed ClickOnce (.NET App) for a company that I was working at broke on certain Windows 10 computers, after some research I found the issue was caused by a Windows 10 Update that broke most ClickOnce installers.

I hope to make some time to research this more later this week or early next week.

**Comment by ConradSollitt** [comment]
Quick update, here is one possible link I found by googling `vscode microsoft certificate revoked`; although it doesn't provide a proper solution.

https://social.technet.microsoft.com/Forums/en-US/07eb3e21-47fe-4441-a842-371931baba8d/certificate-revocation-errors-on-many-msi-files?forum=win10itprosecurity

**Comment by mastion** [comment]
I ran into this today after reformatting my comp. I downloaded it again and got the correct sha. Might be resolved?
![image](https://user-images.githubusercontent.com/774664/90325034-5fad4480-df2b-11ea-924b-a110854dab39.png)

**Comment by ConradSollitt** [comment]
@mastion This issue is computer specific and can probably be closed by @joaomoreno 

If it were more common perhaps docs could be created on the issue.

@renan905  I found this link which shows print screens of a different known certificate showing `Unknown Publisher` and how to fix it. Might be a possible solution for your issue.
https://answers.microsoft.com/en-us/windows/forum/windows_10-security/unblock-an-untrusted-publisher-in-windows-10/e964eea4-d357-4e56-aacc-94088642c7ac

**Comment by joaomoreno** [comment]
This does look indeed like a bad user configuration.

@renan905 What does the Certification Path look like?

![image](https://user-images.githubusercontent.com/22350/91851364-53c9be00-ec5f-11ea-9144-25a057aa05e0.png)
